src/constants/diff-types.js

```diff
@@ -1,16 +1,37 @@
 export const diffTypes = {
-  HIGHLIGHTED_TEXT: 'highlighted text',
-  HIGHLIGHTED_SOURCE: 'highlighted source',
-  HIGLIGHTED_RENDERED: 'higlighted rendered',
-  SIDE_BY_SIDE_RENDERED: 'side-by-side rendered',
-  SIDE_BY_SIDE_TEXT: 'side-by-side text',
-};
-
-// TODO - map each diffType to it's corresponding endpoint string
-export const changeDiffTypes = {
-  [diffTypes.HIGHLIGHTED_TEXT]: 'html_text',
-  // Commented out because this differ has some issues right now
-  // [diffTypes.HIGHLIGHTED_SOURCE]: 'html_source',
-  [diffTypes.HIGHLIGHTED_SOURCE]: 'source',
-  [diffTypes.SIDE_BY_SIDE_RENDERED]: 'html_source'  // HACK
+  HIGHLIGHTED_TEXT: {
+    description: 'highlighted text',
+    diff_service: 'html_text',
```

Comment:
I think we should probably make the `description` field either title or sentence case. I know that’s a little out of scope, but since we are cleaning up how all this is structured…

I also think `diff_service` should be `diffService`. We’re generally going with camel case for all things, with the exception being remote objects. (And maybe we should have the API service translate those… that’s way out of scope here, though.)

Reply:
Changed `description` to be title case. `diff_service` -> `diffService`